main.py

```diff
@@ -7,18 +7,24 @@
 import argparse
 from collections import OrderedDict
 import json
+import os
 from pathlib import Path
 
+from fhir.resources.questionnaire import Questionnaire
+from fhir.resources.valueset import ValueSet
+from fhir.resources.codesystem import CodeSystem
+from fhir.resources import construct_fhir_element
+
 from reproschema.jsonldutils import load_file
 
-from reproschema_to_fhir.config import Config
-from reproschema_to_fhir.fhir import QuestionnaireGenerator
+from src.reproschema_to_fhir.config import Config
+from src.reproschema_to_fhir.fhir import QuestionnaireGenerator
 
 if __name__ == '__main__':
     parser = argparse.ArgumentParser()
-    # TODO: change from hard-coded to command line argument
-    reproschema_folder = Path.cwd() / "reproschema-library/activities/GAD7"
+    parser.add_argument("reproschema_questionnaire")
     args = parser.parse_args()
+    reproschema_folder = Path(str(args.reproschema_questionnaire))
```

Comment:
raise an informative error if this does not exist, also I think the `str` is not necessary